web/src/app/shared/components/tree/tree.component.spec.ts

```diff
@@ -124,7 +146,5 @@ describe('TaskanaTreeComponent', () => {
 
     expect(classification.parentId).toEqual('');
     expect(classification.parentKey).toEqual('');
-    expect(component.switchTaskanaSpinner).toHaveBeenCalledWith(true);
-    expect(component.switchTaskanaSpinner).toHaveBeenCalledWith(false);
```

Comment:
Since I've put the switchTaskanaSpinner(false) statement within the inner subscription I have no idea to test this..